README.md

```diff
@@ -9,17 +9,27 @@
 Documentation available at [Read the Docs](http://mmpy_bot.readthedocs.org/).
 
 
-## What is This
+## Description
 
-A python based chat bot for [Mattermost](http://www.mattermost.org).
+A Python based chat bot for [Mattermost](http://www.mattermost.org). The code for
+this bot was heavily re-factored in v2.0.0 and will only work with Python 3.8 or higher.
```

Comment:
Are we sure it doesn't work with 3.6 or 3.7?

Reply:
I'm fairly certain that `asyncio.create_task` doesn't work or works differently at least in python 3.6, and it's used in a handful of places around the code. If I have time this weekend I'll try to automatically run the unit tests on various Python versions, but for now 3.8 is the only version we can guarantee

Reply:
I am fine with drawing a line in the sand and making 3.8+ the minimum Python version for mmpy_bot 2.x

Reply:
Yep both `asyncio.create_task` and `asyncio_run` were introduced in 3.7, so we can definitely rule out 3.6. We can definitely try out 3.7 at a later time though :+1:

Reply:
Unfortunately it looks like 3.7 is currently not supported since I use `functools.cached_property`, which had not yet been introduced back then. In theory we could extend support to 3.7 by using the third-party `cached-property` package, but let's keep it at 3.8+ for now. If someone really needs 3.7 support we can always fix it in the future.